docs/dev/plugins/cloudFoundry/deploy-cf-bosh-lite.md

```diff
@@ -0,0 +1,161 @@
+  ruby ruby-dev openssl libxslt-dev libxml2-dev \
+  libssl-dev libreadline6 libreadline6-dev \
+  libyaml-dev libsqlite3-dev sqlite3
+```
+
+### Deploy BOSH-Lite
+
+```bash
+git clone https://github.com/cloudfoundry/bosh-deployment ~/workspace/bosh-deployment
+mkdir -p ~/deployments/vbox
+cd ~/deployments/vbox
+bosh create-env ~/workspace/bosh-deployment/bosh.yml \
+  --state ./state.json \
+  -o ~/workspace/bosh-deployment/virtualbox/cpi.yml \
+  -o ~/workspace/bosh-deployment/virtualbox/outbound-network.yml \
+  -o ~/workspace/bosh-deployment/bosh-lite.yml \
+  -o ~/workspace/bosh-deployment/bosh-lite-runc.yml \
+  -o ~/workspace/bosh-deployment/jumpbox-user.yml \
+  --vars-store ./creds.yml \
+  -v director_name="Bosh Lite Director" \
+  -v internal_ip=192.168.50.6 \
+  -v internal_gw=192.168.50.1 \
+  -v internal_cidr=192.168.50.0/24 \
+  -v outbound_network_name=NatNetwork
+```
+
+### Setup BOSH-CLI environment alias
+
+```bash
+osh alias-env vbox -e 192.168.50.6 --ca-cert <(bosh int ./creds.yml --path /director_ssl/ca)
+export BOSH_CLIENT=admin
+export BOSH_CLIENT_SECRET=`bosh int ./creds.yml --path /admin_password`
+```
+
+### Check that it worked
+When executing this command:
+```bash
+bosh -e vbox env
+```
+It should return this (with a different UUID):
+```
+Using environment '192.168.50.6' as client 'admin'
+
+Name      Bosh Lite Director  
+UUID      7dfc1985-06a5-4fa8-aaef-1c049bad5b03  
+Version   264.3.0 (00000000)  
+CPI       warden_cpi  
+Features  compiled_package_cache: disabled  
+          config_server: disabled  
+          dns: disabled  
+          snapshots: disabled  
+User      admin  
+
+Succeeded
+```
+
+### Add IP Route
+
+```bash
+sudo ip route add   10.244.0.0/16 via 192.168.50.6
+```
+
+## Deploing CloudFoundry
+
+### Clone the CF-Deployment Repository
+
+```bash
+cd ~
+git clone https://github.com/cloudfoundry/cf-deployment  
+cd cf-deployment
+```
+
+### Upload the BOSH-Lite Cloud config
+
+This step requires User confirmation (you have to enter `y`)
+
+```bash
+bosh -e vbox update-cloud-config iaas-support/bosh-lite/cloud-config.yml
+```
+
+### Upload the BOSH-Lite Stemcell
+
+```bash
+bosh -e vbox upload-stemcell https://bosh.io/d/stemcells/bosh-warden-boshlite-ubuntu-trusty-go_agent
+```
+
+### Deploy CloudFoundry
+
+Be prepared to wait here. This takes very long (~1-2 Hours)
+
+This step requires User confirmation (you have to enter `y`) after several minutes.
+The Longer part comes after the confirmation.
+
+```bash
+bosh -e vbox -d cf deploy cf-deployment.yml \
+  -o operations/bosh-lite.yml \
+  --vars-store deployment-vars.yml \
+  -v system_domain=bosh-lite.com
+```
+
+## Install CF CLI on the host
+
+```bash
+wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+echo "deb http://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+sudo apt-get update
+sudo apt-get install -y cf-cli
+```
+
```

Comment:
maybe add what you have to do if the vm shuts down
from https://bosh.io/docs/bosh-lite#tips

Reply:
@orferch Maybe you can add this? Since this was your job.